Tasks/TwineAuthenticateV0/package.json

```diff
@@ -18,9 +18,9 @@
     "homepage": "https://github.com/Microsoft/azure-pipelines-tasks#readme",
     "dependencies": {
         "@types/mocha": "^5.2.7",
-        "@types/node": "^10.17.0",
+        "@types/node": "^16.11.39",
         "@types/uuid": "^8.3.0",
-        "azure-pipelines-task-lib": "^3.1.7",
+        "azure-pipelines-task-lib": "^4.0.1-preview",
         "ini": "^1.3.6",
         "packaging-common": "file:../../_build/Tasks/Common/packaging-common-1.0.1.tgz",
```

Comment:
Does it work? I see that you didn't change it.. And the same thing on line 27

Reply:
actually, it works, I tested it on Azure as well. But for to be sure I asked the code owner